core/test/unit/apps_spec.js

```diff
@@ -1,526 +0,0 @@
-var path         = require('path'),
```

Comment:
this file i have created four different specs for proxy, permissions, sandbox and dependencies (see `core/test/unit/apps`.